.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,9 @@
-**This project uses [semver](http://semver.org), please check the scope of this pr:**
+### This project uses [semver](semver.org), please check the scope of this pr:
```

Comment:
Agreed though I'm not concerned with making that determination for the sake of this PR.  Consistency between the repos will be addressed in a separate effort.

(if there was an obvious pattern and this deviated, yes - it should be fixed now.  but to use sidetrack this PR to determine a desired pattern is not anything we need to do right now)